helm/kong-app/CHANGELOG.md

```diff
@@ -2,7 +2,44 @@
 
 ## Unreleased
 
-Nothing yet.
+### Changes
+
+* Updated handling of `session_conf_secret` to accommodate Kong 3.6.
```

Comment:
Now that I see it, technically we're syncing ahead of 2.38.0

I suggest changing the entries in CHANGELOG.md and README.md to point to the commit you've synced to

Reply:
Yea, I think I realized that when checking for the CRD changelogs. I'll make the changes.